packages/node/src/indexer/dictionary/v2/utils.ts

```diff
@@ -177,7 +180,16 @@ export function rawFatBlockToEthBlock(block: RawEthFatBlock): EthereumBlock {
       formatTransaction(tx, ethBlock),
     );
 
-    return ethBlock;
+    return {
+      block: ethBlock,
+      getHeader: () => {
+        return {
+          hash: block.Header.hash,
+          height: Number(block.Header.number),
```

Comment:
All these number conversions should be from hex values `BigNumber.from(block.header.number).toNumber()`